CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+- Reduce memory usage of antctl when collecting supportbundle. ([#2813](https://github.com/antrea-io/antrea/pull/2813), [@tnqn])
+- Do not perform SNAT for egress traffic to Kubernetes Node IPs. ([#2762](https://github.com/antrea-io/antrea/pull/2762), [@leonstack])
+- Change flow exporter export expiry mechanism to priority queue based, to reduce CPU usage and memory footprint. ([#2360](https://github.com/antrea-io/antrea/pull/2360), [@heanlan])
+- Send gratuitous ARP for EgressIP via the transport interface, as opposed to the interface with Node IP (if they are different). ([#2845](https://github.com/antrea-io/antrea/pull/2845), [@Jexf])
+- Wait for AntreaProxy to be ready before accessing Service if `antreaProxy.proxyAll` is enabled, to avoid connection issues on Agent startup. ([#2858](https://github.com/antrea-io/antrea/pull/2858), [@tnqn])
+- Ignore hostNetwork Pods selected by Egress, as they are not supported. ([#2851](https://github.com/antrea-io/antrea/pull/2851), [@Jexf])
+- Add windows task delay up to 30s to improve job resiliency (documentation) ([#2864](https://github.com/antrea-io/antrea/pull/2864), [@perithompson]) [Windows]
+- Remove offensive words from scripts and documentation. ([#2799](https://github.com/antrea-io/antrea/pull/2799), [@xiaoxiaobaba])
+- Make Pod labels optional in the flow records. By default, they will not be included in the flow records. Use the `recordContents.podLabels` configuration parameter for the Flow Aggregator to include them. ([#2739](https://github.com/antrea-io/antrea/pull/2739), [@yanjunz97])
+- Use readable name for OpenFlow tables. ([#2585](https://github.com/antrea-io/antrea/pull/2585), [@wenyingd])
+- Improve the OpenAPI schema for CRDs to validate the `matchExpressions` field. ([#2887](https://github.com/antrea-io/antrea/pull/2887), [@wenqiq])
+- Avoid duplicate processing of Egress. ([#2884](https://github.com/antrea-io/antrea/pull/2884), [@Jexf])
+- Fail fast if the source Pod for non-live-traffic Traceflow is invalid. ([#2736](https://github.com/antrea-io/antrea/pull/2736), [@gran-vmv])
+- Update [OVS pipeline documentation] to include information about AntreaProxy. ([#2725](https://github.com/antrea-io/antrea/pull/2725), [@hongliangl])
+- Ignore the IPs of kube-ipvs0 for Egress as they cannot be used for SNAT. ([#2930](https://github.com/antrea-io/antrea/pull/2930), [@Jexf])
+
+### Fixed
+
+- Fix nil pointer error when antrea-agent updates OpenFlow priorities of Antrea-native policies without Service ports. ([#2730](https://github.com/antrea-io/antrea/pull/2730), [@wenyingd])
+- Fix Agent crash when removing the existing NetNat. ([#2751](https://github.com/antrea-io/antrea/pull/2751), [@wenyingd]) [Windows]
+- Fix panic in the Antrea Controller when it processes ClusterGroups that are used by multiple ClusterNetworkPolicies. ([#2768](https://github.com/antrea-io/antrea/pull/2768), [@tnqn])
```

Comment:
How about move all Windows items to be together?

Reply:
done